arcade/hitbox/base.py

```diff
@@ -49,20 +93,36 @@ def __init__(
         self._position = position
         self._scale = scale
 
+        # Per Clepto's testing as of around May 2023, these are better
+        # left uncached because caching them is somehow slower than what
+        # we currently do. Any readers should feel free to retest /
+        # investigate further.
         self._left = None
         self._right = None
         self._bottom = None
         self._top = None
```

Comment:
Are these attributes ever set to non-`None`?  Should we remove them entirely?

Reply:
To my understanding, they aren't. Which approach seems better?
1. Comment them out as well
2. Delete them and move the note to just above the `left` property

Reply:
I think deleting them is good because it reduces memory usage.